[2024-07-19T14:58:49.839+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-19T14:58:49.926+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.md_currency_d manual__2024-07-19T14:58:30.249061+00:00 [queued]>
[2024-07-19T14:58:49.999+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.md_currency_d manual__2024-07-19T14:58:30.249061+00:00 [queued]>
[2024-07-19T14:58:50.005+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-07-19T14:58:50.110+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): md_currency_d> on 2024-07-19 14:58:30.249061+00:00
[2024-07-19T14:58:50.176+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=1219) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-19T14:58:50.190+0000] {standard_task_runner.py:64} INFO - Started process 1237 to run task
[2024-07-19T14:58:50.214+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'md_currency_d', 'manual__2024-07-19T14:58:30.249061+00:00', '--job-id', '343', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpgugvu_9n']
[2024-07-19T14:58:50.220+0000] {standard_task_runner.py:91} INFO - Job 343: Subtask md_currency_d
[2024-07-19T14:58:51.024+0000] {task_command.py:426} INFO - Running <TaskInstance: insert_data.md_currency_d manual__2024-07-19T14:58:30.249061+00:00 [running]> on host 04b0ec8dd5cd
[2024-07-19T14:58:51.525+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sfakhrutdinov' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='md_currency_d' AIRFLOW_CTX_EXECUTION_DATE='2024-07-19T14:58:30.249061+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-19T14:58:30.249061+00:00'
[2024-07-19T14:58:51.544+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-19T14:58:51.674+0000] {base.py:84} INFO - Using connection ID 'postgres-db' for task execution.
[2024-07-19T14:58:51.933+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-19T14:58:51.954+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-19T14:58:52.020+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=md_currency_d, run_id=manual__2024-07-19T14:58:30.249061+00:00, execution_date=20240719T145830, start_date=20240719T145849, end_date=20240719T145852
[2024-07-19T14:58:52.183+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-19T14:58:52.304+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
